### ISSUE TITLE
Failed to load a custom pack on Lunii device

### ISSUE THREAD

**Issue by aymeric-spiga** [marian-m12l/studio#102]
**Describe the bug**
Everything works fine (official packs being loaded with metadata, custom file seen in the STUdio library and editor), except that when dragging and dropping a custom file (for instance one from the unofficial google spreadsheet) the file does not load on the Lunii

**To Reproduce**
- Open STUdio
- Load a custom file in the library
- Drag and drop to the Lunii part on the left

**Expected behavior**
Should load file on the Lunii, but does not! This is a bit weird (see log below). It does not seem to find a particular method in the Lunii library. The jar files of those libraries are correctly positioned in the `.studio/lib` folder

**Logs**
```
2020-05-24 19:24:08,493 [WARN ] [s.w.s.LibraryService]: Pack to transfer is in archive format. Converting to binary format and storing in temporary file: /tmp/yorgl.zip9124571790389773865.pack
2020-05-24 19:24:08,499 [WARN ] [s.w.s.LibraryService]: Reading archive format pack
2020-05-24 19:24:09,126 [WARN ] [s.w.s.LibraryService]: Uncompressing pack assets
2020-05-24 19:24:27,260 [WARN ] [s.w.s.LibraryService]: Writing binary format pack
2020-05-24 19:24:31,794 [INFO ] [c.l.d.g.r.AbstractUsb4JavaLuniiDriver]: Get handler handle for handler : Usb4JavaLuniiDevice{path=[1, 0, 0, 0, 0, 0, 1], busNumber=1, portNumber=1, deviceAddress=5, deviceHandle=libusb device handle 0x0, handler=libusb device 0x7f134025b190}
2020-05-24 19:24:31,798 [INFO ] [c.l.d.g.r.AbstractUsb4JavaLuniiDriver]: Claim interface for handler : Usb4JavaLuniiDevice{path=[1, 0, 0, 0, 0, 0, 1], busNumber=1, portNumber=1, deviceAddress=5, deviceHandle=libusb device handle 0x7f132c000bf0, handler=libusb device 0x7f134025b190}
2020-05-24 19:24:31,809 [INFO ] [c.l.d.g.r.AbstractUsb4JavaLuniiDriver]: Release interface for handler : Usb4JavaLuniiDevice{path=[1, 0, 0, 0, 0, 0, 1], busNumber=1, portNumber=1, deviceAddress=5, deviceHandle=libusb device handle 0x7f132c000bf0, handler=libusb device 0x7f134025b190}
2020-05-24 19:24:31,809 [INFO ] [c.l.d.g.r.AbstractUsb4JavaLuniiDriver]: Dispose handler : Usb4JavaLuniiDevice{path=[1, 0, 0, 0, 0, 0, 1], busNumber=1, portNumber=1, deviceAddress=5, deviceHandle=libusb device handle 0x7f132c000bf0, handler=libusb device 0x7f134025b190}
2020-05-24 19:24:31,810 [INFO ] [s.w.s.StoryTellerService]: Computing pack start sector...
2020-05-24 19:24:31,814 [ERROR] [s.w.s.StoryTellerService]: Failed to add pack to device
java.lang.NoSuchMethodException: com.lunii.device.wrapper.raw.RawVirtualContent.addPack(java.util.UUID,short,int,short)
	at java.base/java.lang.Class.getDeclaredMethod(Class.java:2553)
	at studio.webui.service.StoryTellerService$3.run(StoryTellerService.java:175)
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
	at java.base/java.util.TimerThread.run(Timer.java:506)
java.lang.NoSuchMethodException: com.lunii.device.wrapper.raw.RawVirtualContent.addPack(java.util.UUID,short,int,short)
	at java.base/java.lang.Class.getDeclaredMethod(Class.java:2553)
	at studio.webui.service.StoryTellerService$3.run(StoryTellerService.java:175)
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
	at java.base/java.util.TimerThread.run(Timer.java:506)
```

**Desktop and environment (please complete the following information):**
 - OS: Linux x86_64
 - Browser Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:76.0) Gecko/20100101 Firefox/76.0
 - Java version:
```
java version "14.0.1" 2020-04-14
Java(TM) SE Runtime Environment (build 14.0.1+7)
Java HotSpot(TM) 64-Bit Server VM (build 14.0.1+7, mixed mode, sharing)
```
 - STUdio application version: 0.1.15

**Additional context**
With the latest version of the Luniistore (updated today). Maybe the problem is coming from a recent update in the proprietary library.

By the way, merci beaucoup pour le super boulot

**Comment by aymeric-spiga** [comment]
PS: problem reproduced both in Firefox and Chromium, does not seem to be an issue with web browser

**Comment by marian-m12l** [comment]
Hi,

Thanks for letting me know!

This is fixed in release 0.1.16, since it was a tiny change in the method signature. Hopefully the last fix of this kind, before the custom driver takes over...

**Comment by aymeric-spiga** [comment]
works like a charm, now! merci beaucoup, bravo. I may beta-test the version with the custom driver in the following days.
